frontend/src/components/LocationPopOut/LocationPopOut.tsx

```diff
@@ -9,32 +9,52 @@ import {
   StyledMediaBox,
   StyledTooltip
 } from '../design/StyledLocationPopOut';
-import icon from '../../static/pin-308802.svg';
+import verifiedIcon from '../../static/verified-icon.svg';
+import unverifiedIcon from '../../static/unverified-icon.svg';
+import waitingIcon from '../../static/waiting-icon.svg';
 import img from '../../static/image-not-found.jpg';
+import { verificationStatusTypes } from '../../static/verificationStatusTypes';
```

Comment:
For types we have separate file types.ts at the root of the folder front-end. I think this data related to types data and could be in types.ts